uiowa/uiowa
# Migration physics

Here is the migration module for the Physics and Astronomy news items. Will solve #3369.

Still to do:

- [x] Properly handle lookup and storage of tags from old site into existing taxonomy (tags).
- [x] If physics_tags.yml file is not needed for creation of taxonomy terms, can delete.

# How to test

- In DevDesktop, clone site and pull database from uiclas application (may not be able to do)
- git checkout migration_physics (and the usual stuff)
- blt ds --site=physics.uiowa.edu
- drush @physics.local uli
- Enable "Physics Migrate" module in Extend
- Go to Configuration > System > SiteNow Migrate
- Enter default values from comments except for Database name (physics_uiowa_edu) and Base URL (https://physics.uiowa.edu).
- Go to Adminsitration > Structure > Migrations > Edit SiteNow migration group.
- Execute Articles migration (should be at least 334 items).
